app/src/backend/queue.ts

```diff
@@ -16,7 +16,7 @@ const PRIORITY = {
 /**
  * @todo Better exposure of debugging.
  */
-const enableDebug = false;
+const enableDebug = true;
```

Comment:
Should this be false?

Reply:
O yes it should. They slipped in while I was committing